packages/cli/src/config/config.ts

```diff
@@ -247,6 +247,10 @@ export async function loadCliConfig(
       (v) => v === 'true' || v === '1',
     );
 
+  if (process.env.GEMINI_CLI_SYSTEM_SETTINGS_PATH) {
```

Comment:
having this method have a side effect other than returning a Config is unfortunate. I would suggest we instead have settings.ts read process.env.GEMINI_CLI_SYSTEM_SETTINGS_PATH
directly.